x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -132,14 +134,20 @@ API key information.
 <2> optional expiration in milliseconds for this API key
 <3> generated API key
 
-The API key returned by this API can then be used by sending a request with an
-`Authorization` header with a value having the prefix `ApiKey` followed
-by the _credentials_, where _credentials_ are the base64 encoding of `id` and `api_key` joined by a colon.
+To create your credentials, encode the returned `id:api_key` in base64. Ensure
+the value does not include a trailing newline.
```

Comment:
I think where `id:api_key` comes from is not clear. Also the term "credentails" is introduced with no context. Overall I prefer the original version. 

I think what we really want to say here is just the fact: when creating the credentials using `echo`, we must be careful to not include a trailing newline (echo's default behaviour). This is just one way of creating the credentials and we should be explicit about it. So I'd suggest we keep the original text and add a new sentence like:
```
One way to create the _credentails_ from CLI on an Unix-like system is as the follows:
[source,shell]
--------------------------------------------------
# Please note the use of "-n" to instruct echo to not print the trailing newline character.
# It should return VnVhQ2ZHY0JDZGJrUW0tZTVhT3g6dWkybHAyYXhUTm1zeWFrdzl0dk5udw==
echo -n "VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw" | base64
--------------------------------------------------
```